env0/resource_aws_credentials.go

```diff
@@ -38,6 +38,7 @@ func resourceAwsCredentials() *schema.Resource {
 				ForceNew:      true,
 				ConflictsWith: []string{"access_key_id"},
 				RequiredWith:  []string{"arn"},
+				Deprecated:    "field will be removed in the near future",
```

Comment:
this field should also become optional, don't we need to remove
`RequiredWith: []string{"arn"}` ?

Reply:
sure I can do that.

Reply:
removed the "requiredWith".
for costs I added optional and removed required.